docs/versioned_docs/version-v0.8/concepts/email-sms.md

```diff
@@ -3,8 +3,8 @@ id: email-sms
 title: Out-of-band communication via E-Mail and SMS
```

Comment:
Please move this to docs/docs or it will be overwritten by the next automation step :)

Reply:
Ah ok, I think the two documents did not match fully so I wasn't sure whether you might have copied something into this doc that should have been in the other doc (as this will be overwritten). If the docs/concepts/email-sms guide is up to date then you can ignore this comment :)

Reply:
Yeah there's no diff :)